packages/template/base/tmpl/_appveyor.yml

```diff
@@ -8,7 +8,7 @@ cache:
   - node_modules
 branches:
   only:
-    - master
+    - main
```

Comment:
thought: this change really doesn't matter, this file should be removed in some way or another before we release to stable.